### ISSUE TITLE
Additional options for midi, exports

### ISSUE THREAD

**Issue by quattj** [sykhro/vgmtrans-qt#6]
I am in the process of creating midis from Super Nintendo (and a few other systems) games.  Currently, vgmtrans assigns instruments for an exported MIDI file in numerical order based on the order of the instruments in the SPC file.  This sounds good when the MIDI is played using the extracted soundfont, but terrible otherwise.  I'd like to request an option to manually select which General MIDI instruments are assigned to which in-game instruments.

For example, the game track has a church organ.  Allow me to assign GM instrument 20 Church Organ to that channel on export (instead of the next available low numbered instrument).  
The game track has timpani.  Allow me to assign GM instrument 48 Timpani to that channel on export.  
This way, the MIDI file will sound decent whether or not it is played back using the extracted soundfont.

Additionally, it would really help if the MIDI file could be played "live" as instruments are selected by the user to hear the changes, similar to how spc2midi works, but with the addition of being able to save a set of defined instruments so they may be loaded and applied to other files you open.

Following the example above, in-game Church Organ gets assigned to GM 20, in-game Timpani gets assigned to GM 48.  Allow save of MIDI instrument set.  Then when the next track of the game is loaded to convert, allow loading and applying previous instrument settings to the current track.

**Comment by AlbertV97** [comment]
I agree with this, I also export midis a lot! and the way how they come out is not accurate
